src/store/useProjectStore.ts

```diff
@@ -153,53 +153,33 @@ const store: StateCreator<ProjectStore> = (set, get) => ({
 		return { data: akProjectArchiveZip, error: undefined };
 	},
 
-	createProjectFromManifest: async (projectManifest: string) => {
+	createProjectFromManifest: async (projectManifest) => {
 		const currentOrganization = useOrganizationStore.getState().currentOrganization;
-
-		const { data: newProjectId, error } = await ProjectsService.createFromManifest(
-			projectManifest,
-			currentOrganization?.id
-		);
-
-		if (error) {
-			return { data: undefined, error };
-		}
-
-		if (!newProjectId) {
-			return { data: undefined, error: t("projectCreationFailed", { ns: "errors" }) };
-		}
-
 		const manifestObject = load(projectManifest) as {
 			project?: { name: string };
 		};
 
-		let projectName = manifestObject?.project?.name;
-
+		const projectName = manifestObject?.project?.name;
 		if (!projectName) {
-			const { data: project, error: getProjectError } = await ProjectsService.get(newProjectId);
-
-			if (!getProjectError) {
-				return { data: undefined, error: getProjectError };
-			}
-			if (!project) {
-				return { data: undefined, error: t("projectLoadingFailed", { ns: "errors" }) };
-			}
-
-			projectName = project.name;
+			return { data: undefined, error: new Error(t("projectNameRequiredManifest", { ns: "errors" })) };
```

Comment:
NIT PICK: Please move this translation from "errors" namespace to "stores.projects" or "stores.templates" or "your-suggestion".